webapp/src/components/MeetingQuickView.tsx

```diff
@@ -128,16 +128,16 @@ const MeetingQuickView = ({ meeting }: MeetingQuickViewProps) => {
               <React.Fragment key={meetingNote.id}>
                 {(index === 0 ||
                   meetingNote.agenda_owning_participant_id !==
-                    meetingNotes[index - 1].agenda_owning_participant_id) && (
-                  <Typography sx={{ fontWeight: 'bold' }}>
-                    {meetingNote.agenda_owning_participant_id === null
-                      ? 'Action items'
-                      : meetingNote.agenda_owning_participant_id ===
-                        currentParticipantId
-                      ? 'My agenda items'
-                      : 'Their agenda items'}
-                  </Typography>
-                )}
+                  meetingNotes[index - 1].agenda_owning_participant_id) && (
+                    <Typography sx={{ fontWeight: 'bold' }}>
+                      {meetingNote.agenda_owning_participant_id === null
+                        ? 'Action items'
+                        : meetingNote.agenda_owning_participant_id ===
+                          currentParticipantId
+                          ? 'My agenda items'
+                          : 'Their agenda items'}
+                    </Typography>
+                  )}
```

Comment:
Not sure why the indentation is changing

Reply:
Thanks for bringing it up. 
Done.
![image](https://user-images.githubusercontent.com/68254832/175380309-fb3db14e-abd3-4ff1-806a-e4e6c2f60224.png)